app/utils/analytics.js

```diff
@@ -24,6 +24,7 @@ exports.withAnalytics = (charge, param) => {
       analyticsId: charge.gatewayAccount.analyticsId,
       type: charge.gatewayAccount.type,
       paymentProvider: charge.gatewayAccount.paymentProvider,
-      amount: charge.amount
+      amount: charge.amount,
+      testingVariant: 'original'
```

Comment:
is this supposed to be hard coded?

Reply:
default value that is overwritten by the `_merge`